MediaBrowser.Providers/TV/TvdbExternalId.cs

```diff
@@ -0,0 +1,23 @@
+﻿using MediaBrowser.Controller.Entities.TV;
+using MediaBrowser.Controller.Providers;
+using MediaBrowser.Model.Entities;
+using MediaBrowser.Providers.Plugins.TheTvdb;
+
+namespace MediaBrowser.Providers.TV
+{
+    public class TvdbExternalId : IExternalId
```

Comment:
Missing documentation here

Reply:
Strange. this was fixed in a commit from yesterday.

Reply:
Added in latest commit. I could've sworn it was there before.